Portfolio/index.html

```diff
@@ -1,16 +1,65 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
```

Comment:
The 'top' attribute should be specified with a CSS property such as 'position: absolute; top: 0;' instead of being used as a standalone attribute on a div element.